rocoto/WaterInTheWest.sh

```diff
@@ -10,9 +10,13 @@ EXPT=$(basename ${HOMEhafs})
 SUBEXPT=$EXPT
 conf=../parm/WaterInTheWest.conf
 opts="-t -f"
+
+# To enable graphics, uncomment this. But only on Hera!
```

Comment:
Can there be a check for this ... `if hera`

Reply:
That script doesn't know if it is on Hera. With a bit of work, I could have run_hafs.py override the run_pygraf if you're on GAEA.